src/activation.rs

```diff
@@ -121,14 +123,20 @@ pub fn receive_descriptors(unset_env: bool) -> Result<Vec<FileDescriptor>, SdErr
 
 /// Check for named file descriptors passed by systemd.
 ///
-/// Like `sd_listen_fds`, but this will also return a vector of names associated with each file
-/// descriptor.
+/// Like `receive_descriptors`, but this will also return a vector of names
+/// associated with each file descriptor.
 pub fn receive_descriptors_with_names(
     unset_env: bool,
 ) -> Result<Vec<(FileDescriptor, String)>, SdError> {
     let pid = env::var("LISTEN_PID");
     let fds = env::var("LISTEN_FDS");
     let names = env::var("LISTEN_FDNAMES");
+    log::trace!(
+        "LISTEN_PID = {:?}; LISTEN_FDS = {:?}; LISTEN_FDNAMES = {:?}",
```

Comment:
I wonder if one line per variable would be better :)

Reply:
Possibly, but I don't expect much output from this. Longer term idea is to use a single log entry with [structured fields](https://docs.rs/log/0.4.8/log/kv/index.html) when that is stabilized.

Yes, there a few lines of code duplication, but I've had the de-duplicated version too in the past and it didn't help readability, so I'll stick to this. (I know this is a bit of subjective taste, sorry :)